packages/number-line/src/number-line/index.jsx

```diff
@@ -188,8 +188,11 @@ export class NumberLine extends React.Component {
       ? getCorrectAnswerElements()
       : getAnswerElements();
 
-    let maxPointsMessage = () =>
-      `You can only add ${model.graph.maxNumberOfPoints} elements`;
+    let maxPointsMessage = () => {
+      let plural;
+      model.graph.maxNumberOfPoints !== 1 ? plural = 's' : plural = '';
```

Comment:
I would rather add an if / else statement OR just add the check into the return line.

Reply:
ok